.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
And devcontainer.json.

```
// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
// https://github.com/microsoft/vscode-dev-containers/tree/v0.202.5/containers/ubuntu
{
	"name": "Ubuntu",
	"runArgs": ["--init"],
	"build": {
		"dockerfile": "Dockerfile",
		// Update 'VARIANT' to pick an Ubuntu version: hirsute, focal, bionic
		// Use hirsute or bionic on local arm64/Apple Silicon.
		"args": { "VARIANT": "focal" }
	},

	// Set *default* container specific settings.json values on container create.
	"settings": {},


	// Add the IDs of extensions you want installed when the container is created.
	"extensions": [],

	// Use 'forwardPorts' to make a list of ports inside the container available locally.
	// "forwardPorts": [],

	// Use 'postCreateCommand' to run commands after the container is created.
	// "postCreateCommand": "uname -a",

	// Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
	"remoteUser": "vscode",
	"features": {
		"terraform": "latest",
		"azure-cli": "latest"
	}
}
```

So I'm not sure if we want to change it now, or later? And not sure which devcontaienr we should use. There are many options (C#, etc)
![image](https://user-images.githubusercontent.com/25376553/141084415-fe2a0eb8-387b-4e87-b022-893ffc2b5aee.png)

Reply:
Improvements to the dev container will be handled on a separate branch.